.vscode/extensions.json

```diff
@@ -4,7 +4,6 @@
     "bradlc.vscode-tailwindcss",
     "charliermarsh.ruff",
     "esbenp.prettier-vscode",
-    "heybourn.headwind",
```

Comment:
Nice, thanks for this!

Reply:
We need to also remove this from the readme and contributing guide as well, but I'll get those 😊 There's a dropdown with suggested extensions in each :)

Reply:
@andrewtavis I can make a quick update for the readme and contributing and update this PR. I forgot to check for any headwind mentions.

Reply:
Sure thing, thanks @lucianmurmurache! I'm halfway through 🥴😅

Reply:
✔️